### PR TITLE
Added a new paragraph in the contribution guide about an error because it occurred to me and may occur to others also.

### PR DESCRIPTION
**Pull request by ibilalkayy**
**What this PR does / why we need it**:

This PR contains a paragraph in which there is a remember guide when running the `make docs` command. For some users, it may give an error because they have not added the `/tmp` path into the Docker settings. Without this setting, running the `http://localhost:3002/docs/loki/latest/` URL won't work.

That's why I added this guide so it becomes easy for others also.

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [x] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
  - [ ] If the change is worth mentioning in the release notes, add `add-to-release-notes` label
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

File: CONTRIBUTING.md
Comment (jdbaldry) [inline CONTRIBUTING.md:157]: I might be tempted to only document adding the parent temporary directory (or the value of TMPDIR if it is set) because the script creates different subdirectories each time.

```suggestion
Then you can go to Docker Desktop settings and open the resources, add the temporary directory path `/tmp`.
```

**Review by jdbaldry — COMMENTED** [review]
Thanks for the contribution, I think this is helpful advice!

The script uses `mktemp` to create the temporary file so it might be worth noting that you can change the directory in which the temporary file is made using the `TMPDIR` environment variable.

I'm not a maintainer on these files so I'll let one of those do an additional review.

**Review by MichelHollands — APPROVED** [review]
LGTM

**Pull request merged**
